en/enable-tls-for-dm.md

```diff
@@ -0,0 +1,620 @@
+
+        After the object is created, `cert-manager` generates a `${cluster_name}-cluster-client-secret` Secret object to be used by the clients of the DM components.
+
+### Deploy the DM cluster
+
+When you deploy a DM cluster, you can enable TLS between DM components, and set the `cert-allowed-cn` configuration item to verify the CN (Common Name) of each component's certificate.
+
+> **Note:**
+>
+> Currently, you can set only one value for the `cert-allowed-cn` configuration item of DM-master. Therefore, the `commonName` of all `Certificate` objects must be the same.
+
+- Create the `dm-cluster.yaml` file:
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: DMCluster
+metadata:
+  name: ${cluster_name}
+  namespace: ${namespace}
+spec:
+  tlsCluster:
+    enabled: true
+  version: v2.0.0-rc.2
+  pvReclaimPolicy: Retain
+  discovery:
+    address: "http://${tidb_cluster_name}-discovery.${tidb_namespace}:10261"
+  master:
+    baseImage: pingcap/dm
+    replicas: 1
+    storageSize: "1Gi"
+    config:
+      cert-allowed-cn:
+        - TiDB
+  worker:
+    baseImage: pingcap/dm
+    replicas: 1
+    storageSize: "1Gi"
+    config:
+      cert-allowed-cn:
+        - TiDB
+```
+
+Use the `kubectl apply -f dm-cluster.yaml` file to create a DM cluster.
+
+### Configure `dmctl` and connect to the cluster
+
+Get into the DM-master Pod:
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl exec -it ${cluster_name}-dm-master-0 -n ${namespace} sh
+```
+
+Use `dmctl`:
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+cd /var/lib/dm-master-tls
+/dmctl --ssl-ca=ca.crt --ssl-cert=tls.crt --ssl-key=tls.key --master-addr https://127.0.0.1:8261 list-member
+```
+
+## Use DM to migrate data between MySQL/TiDB databases that enable TLS for the MySQL client
+
+The following section describes how to configure DM to enable synchronously TLS-authenticated MySQL/TiDB database of MySQL client.
+
+To learn how to enable TLS for the MySQL client of TiDB, refer to [Enable TLS for the MySQL Client](enable-tls-for-mysql-client.md).
+
+### Step 1: Create the Kubernetes Secret object of TLS of each MySQL client
+
+Suppose you deployed a MySQL/TiDB database with MySQL client TLS enabled. To create a Secret object for the TiDB cluster using the following command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl create secret generic ${mysql_secret_name1} --namespace=${namespace} --from-file=tls.crt=client.pem --from-file=tls.key=client-key.pem --from-file=ca.crt=ca.pem
+kubectl create secret generic ${tidb_secret_name} --namespace=${namespace} --from-file=tls.crt=client.pem --from-file=tls.key=client-key.pem --from-file=ca.crt=ca.pem
+```
+
+### Step 2: Mount secret objects to DM cluster
+
+After creating the Kubernetes Secret object of the upstream and downstream databases, you need to set `spec.tlsClientSecretNames` so that you can mount the Secret object to the Pod of DM-master/DM-worker.
```

Comment:
```suggestion
After creating the Kubernetes Secret objects for the upstream and downstream databases, you need to set `spec.tlsClientSecretNames` so that you can mount the Secret objects to the Pod of DM-master/DM-worker.
```